src/common/form/Form.js

```diff
@@ -53,9 +53,10 @@ const form = ({ validate }: FormSetupProps = {}): Function => <
       try {
         const { onSubmit } = this.props;
         this._formStore.setSubmitting(true);
+
         const result = callback.call
-          ? callback(this._formStore.values)
-          : onSubmit && onSubmit(this._formStore.values);
+          ? callback(this._formStore.submittingValues)
```

Comment:
I don't really like `submittingValues` -- `formValues` might be better.

Reply:
`formStore.values` are form values, but submittingValues are translated before submit values. ..its `lIke` result of our DAO.translator.toApi, but on the form level.

Reply:
Ok, that makes sense. I didn't realize this was calculated when I wrote the comment.